### PR TITLE
Thuy: Prompt 2.3: Fix the web content tags

### PR DESCRIPTION
**Pull request by thuycao-gs**
Hello @huynguyen-codeengine,
This is my PR for prompt 2.3.
Could you please consider and let me know if it has any problem?
Thanks.

Steps to reproduce
1. Web Content > Add Basic Web Content
2. Copy and paste these tags into the Tags field:
"Redriven,carburized,renotarizing,bumph,englewood,unprotracted,avower,preincorporated,tungsten,sewerage,eulogized,preimpress,vaporetto,provinciality.,Subformation,verbose,epochal,porkpie,haematoid,anticonformity,skagerrak,balding,helot,captor,bwg,repropose,rainlessness,dumps.,Rooter,punishable,florry,overillustrative,nonwatertight,coachwork,consecration,caryophyllaceous,catlin,propublication,jon,miaul,anthophilous,frigidity.,Deracinnating,qualitatively,lanarkshire,breed,ululant,injudicious,scorpionfish,oestrogen,kur,preeternal,noncrystallizable,retardate,sailor,decuple.,Recce,bollix,regeneration,manistee,haematocyte,cynwulf,immaterialise,centricity,stinkeroo,domelike,bighorn,pillarlike,mercaptide,delicious"
3. Press "enter" to add Tags
4. Publish Web Content. Successful publish.
5. Create another web content with the same tags.
6. Publish Web Content

Expected Results: Successful publish.

Actual Results: Your request failed to complete.

### CODE REVIEW

**Comment by huynguyen-codeengine** [comment]
Failed source formatter.

ant setup-sdk compile
cd portal-impl
ant format-source-latest-author

**Comment by thuycao99** [comment]
> Failed source formatter.
> 
> ant setup-sdk compile
> cd portal-impl
> ant format-source-latest-author

File: portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java [hunk 1]
```diff
@@ -122,7 +122,8 @@ public AssetTag addTag(
 		List<AssetTag> tags = new ArrayList<>();
 
 		for (String name : names) {
-			AssetTag tag = fetchTag(group.getGroupId(), name);
+			String nameValid = StringUtil.toLowerCase(name.replaceAll("\\s+", ""));
```
Comment (huynguyen-codeengine) [inline portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java:125]: not a proper way to trim string

Comment (thuycao-ces) [reply portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java]: I updated it. Thank you.

File: portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java [hunk 2]
```diff
@@ -122,7 +122,9 @@ public AssetTag addTag(
 		List<AssetTag> tags = new ArrayList<>();
 
 		for (String name : names) {
-			AssetTag tag = fetchTag(group.getGroupId(), name);
+
+			String nameValid = StringUtil.toLowerCase(name.replaceAll("\\s+",""));
```
Comment (huynguyen-codeengine) [inline portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java]: Use StringUtil to trim string

Comment (thuycao-ces) [reply portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java]: I updated it. Please check it again.
Thanks.

**Pull request closed without merging**